doc/api/fs.md

```diff
@@ -8,21 +8,25 @@
 
 <!-- source_link=lib/fs.js -->
 
-The `fs` module provides an API for interacting with the file system in a
-manner closely modeled around standard POSIX functions.
+The `fs` module allows interacting with the file system in a
+way closely modeled around standard POSIX functions.
 
 To use this module:
 
 ```js
 const fs = require('fs');
 ```
 
-All file system operations have synchronous and asynchronous forms.
+All file system operations have synchronous, asynchronous, and Promised-based
```

Comment:
This addition seems a little strange to me, it makes it sound like the Promise-based API utilizes something other than sync or async processing.

Reply:
```suggestion
All file system operations have synchronous, asynchronous, and Promise-based
```